core/base/v7/inc/ROOT/TLogger.hxx

```diff
@@ -113,8 +113,16 @@ public:
 } // namespace Experimental
 } // namespace ROOT
 
+#if defined(_MSC_VER)
+#define LOG_PRETTY_FUNCTION __FUNCSIG__
+#elif defined(__GNUC__) || defined(__clang__)
```

Comment:
I'd rather leave this the default: it's more likely that we miss a compiler here (eg ICC) than that compiler not supporting `__PRETTY_FUNCTION__`... Ie can you remove the if here and the func part below? We will immediately see if a compiler does not support this. Thanks!

Reply:
Done